zerver/actions/user_groups.py

```diff
@@ -386,7 +391,10 @@ def add_subgroups_to_user_group(
 
 @transaction.atomic
 def remove_subgroups_from_user_group(
-    user_group: UserGroup, subgroups: List[UserGroup], *, acting_user: Optional[UserProfile]
+    user_group: NamedUserGroup,
```

Comment:
We would add subgroups to a `UserGroup` object as well but we would probably do it in a separate function as we might not create the `RealmAuditLog` entries for them and also not send events for them.

Reply:
Yeah, that makes sense.